### PR TITLE
Styling of Form Page

### PR DESCRIPTION
**Pull request by acushlakoncept**
The style.css file was modified as thus:

- Outline style was added to the form input box

- Background page color was changed

- The icons size at the top of the form was increased

- Links color was changed to suit the original intuit page

- Screenshot image of the Readme file was updated

- Lint errors were fixed in the style.css file

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @acushlakoncept and @emmanuellekamwa, you did a great job with this project :+1: It is not a good practice to switch between PR or create a different PR  for a single project. This makes tracking the comments from the previous TSE harder for us. I would advise you to use only 1 PR for a single project. :smile: However, let us address the comment below before we proceed:
- Kindly avoid using ```ids```  and ```tag``` names for styling, stylings are done by using classes only
- Let us implement the DRY rule. Check this [resource](https://speckyboy.com/good-bad-css-practices/)
- Let us use the background in the original page, we could check it by ```ctrl+shift+i``` and clicking on the element's tag and checking for the background-color
sh

Kindly make these changes and submit another code review :smile: 
_Happy coding_ :man_technologist: 
Daniel Larbi Addo on slack

**Comment by acushlakoncept** [comment]
> ## Status: Require changes ♻️ 🚥
> Hi @acushlakoncept and @emmanuellekamwa, you did a great job with this project 👍 It is not a good practice to switch between PR or create a different PR for a single project. This makes tracking the comments from the previous TSE harder for us. I would advise you to use only 1 PR for a single project. 😄 However, let us address the comment below before we proceed:
> 
> * Kindly avoid using `ids`  and `tag` names for styling, stylings are done by using classes only
> * Let us implement the DRY rule. Check this [resource](https://speckyboy.com/good-bad-css-practices/)
> * Let us use the background in the original page, we could check it by `ctrl+shift+i` and clicking on the element's tag and checking for the background-color
>   sh
> 
> Kindly make these changes and submit another code review 😄
> _Happy coding_ 👨‍💻
> Daniel Larbi Addo on slack

Thank you Daniel for your comment, we will go ahead and implement the DRY principle. However, if you look at one of our [last commits](https://github.com/emmanuellekamwa/html_forms/pull/3/commits/82d67967b281c4f405b884f0e73ade0d8cc7de30), the background color we used is the same as that of the original

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @acushlakoncept and @emmanuellekamwa 👋

You both have done quite well on this project, the layout looks great, and seems to be a good clone of the original webpage.🚀

However, kindly make the following adjustments to get a final thumbs-up:

- [ ] Required - As stated in the previous review please change all ID tags to class selectors. We want to avoid  CSS bad practices, refer to this ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright).

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: style.css [hunk 1]
```diff
@@ -1,143 +1,147 @@
...
-    position: absolute;
-    right: 15px;
+  position: absolute;
+  right: 15px;
 }
 
 #form_body {
-    width: 400px;
-    margin: 20px auto;
-    text-align: center;
```
Comment (mikenath223) [inline style.css]: Kindly change the ID tag selectors, here and in other places on your CSS codebase to class selectors. Class selectors should be defined on your HTML code(class="my-element") and referenced on your CSS code(.my-element {...}).

**Comment by trekab** [comment]
# Approved! :tada: 
Hi @acushlakoncept and @emmanuellekamwa, great job meeting all the requirements for this project! :+1:  You may now merge to master, fill the project completion form, and continue to the next project.

_All the best in your next projects!_
__Treasure Kabareebe__

**Pull request merged**